toolkit-rust/src/lib.rs

```diff
@@ -4,7 +4,7 @@
 //! Tool. A Nexus Tool is a service that can be invoked over HTTP. The Toolkit
 //! automatically generates the necessary endpoints for the Tool.
 //!
-//! See more documentation at <https://github.com/Talus-Network/gitbook-docs/nexus-sdk/toolkit-rust.md>
+//! See more documentation at <https://github.com/Talus-Network/gitbook-docs/blob/staging/nexus-sdk/toolkit-rust.md>
```

Comment:
I probably commented on the previous PR but I'm not sure this should point to `staging`. Either `production` or maybe even better would be to change these to point to https://docs.talus.network/developer-docs/index-1/toolkit-rust